src/tokens/Assets.tsx

```diff
@@ -55,20 +64,39 @@ import { TokenBalance } from 'src/tokens/slice'
 import { getSupportedNetworkIdsForTokenBalances, sortByUsdBalance } from 'src/tokens/utils'
 
 const DEVICE_WIDTH_BREAKPOINT = 340
+const NUM_OF_NFTS_PER_ROW = 2
+
+const nftImageSize =
+  (variables.width - Spacing.Regular16 * (NUM_OF_NFTS_PER_ROW + 1)) / NUM_OF_NFTS_PER_ROW
```

Comment:
per designs looks like the border on edges is 24, and border between images is 16